Content.Client/Construction/ConstructionMenu.cs

```diff
@@ -250,7 +251,7 @@ void OnBuildToggled(BaseButton.ButtonToggledEventArgs args)
                         IsTile = false,
                         PlacementOption = prototype.PlacementMode
                     },
-                    new ConstructionPlacementHijack(prototype, Owner));
+                    new ConstructionPlacementHijack(_systemManager.GetEntitySystem<ConstructionSystem>(), prototype));
```

Comment:
You can do `EntitySystem.Get` rather than inject `_systemManager`.

Reply:
According to @PJB3005 we should still be using dependency injection with the field+attribute, instead of using the `IoCManager.Resolve<>()` service locator anti-pattern.

Reply:
Weird that he didn't stop the PRs that tore out some of the DI in favour of service location then.

Reply:
Either is fine, I don't have a strong preference.

Reply:
I personally like `EntitySystem.Get` in cases where the code isn't doing a large number of calls to the entity system, since it's much simpler. We should probably settle on a particular style, though.

Reply:
Both patterns are currently being used in the code base without preference, so I guess this is resolved with no change?

Reply:
Yeah, the style thing was meant as a future remark. If there's no strong preference for either form at the moment, then this is resolved.